imports/plugins/core/accounts/client/components/email/updateEmail.js

```diff
@@ -0,0 +1,121 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { Meteor } from "meteor/meteor";
+import { i18next } from "/client/api";
+import { Components } from "@reactioncommerce/reaction-components";
+
+class UpdateEmail extends Component {
+  static propTypes = {
+    email: PropTypes.string,
+    uniqueId: PropTypes.string
+  }
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      email: props.email,
+      newEmail: false,
+      showSpinner: false
+    };
+
+    this.handleFieldChange = this.handleFieldChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  componentWillReceiveProps() {
+    this.setState({ showSpinner: false });
+  }
+
+  emailUpdate() {
+    if (this.state.email === this.props.email) {
+      return false;
+    }
+
+    return true;
+  }
+
+  handleFieldChange = (event, value, field) => {
+    this.setState({
+      [field]: value
+    });
+  }
+
+  handleSubmit = (event) => {
+    event.preventDefault;
+
+    this.setState({ showSpinner: true });
+
+    const { email } = this.state;
+
+    Meteor.call("accounts/validation/email", email, false, (result, error) => {
+      if (error.error) {
+        Alerts.toast(i18next.t("accountsUI.error.invalidEmail", { err: error.reaston }), "error");
+        this.setState({ showSpinner: false });
+        return null;
+      }
+
+      Meteor.call("accounts/updateEmailAddress", email, (err) => {
+        if (err) {
+          Alerts.toast(i18next.t("accountsUI.error.emailAlreadyExists", { err: error.message }), "error");
```

Comment:
Wrong `error` object here in the Alert.  Should be `err` because `error` is from the upper scope.